README.md

```diff
@@ -1,5 +1,130 @@
-MSparse
-=======
+![Ginkgo](/assets/logo.png)
 
-MSparse (Manycore Sparse) is a linear algebra library targeting manycore
-architectures.
+[![Build status](https://gitlab.com/ginkgo-project/ginkgo-public-ci/badges/master/build.svg)](https://github.com/ginkgo-project/ginkgo/commits/master)
+[![Coverage](https://codecov.io/gh/ginkgo-project/ginkgo/branch/master/graph/badge.svg)](https://codecov.io/gh/ginkgo-project/ginkgo)
```

Comment:
Both the `pipeline` and the `codecov` are currently listed as `unknown` when looking at this [README.md](https://github.com/ginkgo-project/ginkgo/blob/release/1.0.0/README.md) from the branch `release/1.0.0`. I would assume it is because everything is not yet in the `master` branch, is that correct?
Otherwise, we would need to fix it.

Reply:
Yes, that is purely the case. They will appear when we merge into master, that is why I did that change last.